### PR TITLE
PHP74

### PR DESCRIPTION
**Pull request by dereuromark**
@ehsanmx Please enable some valid smoke tests here

### CODE REVIEW

**Comment by dereuromark** [comment]
Yeah, that will be your ticket to fix up :)

**Comment by ehsanmx** [comment]
hmm but it mentioend ` 'SprykerTest\Shared\Testify\Exception\StoreNotFoundException'` how is this related to creating tests!
and we have some tests here `SprykerTest/Zed/RabbitMq/Business/Model/Exchange/Filter/ExchangeFilterByNameTest.php`

**Comment by dereuromark** [comment]
Please check these things then once you find time. Ticket is created here by @RodinVasily

**Comment by dereuromark** [comment]
Exactly, you need to fix this PR and then once green we can continue.

**Comment by dereuromark** [comment]
@ehsanmx Whats the status on this?

File: .travis.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@ language: php
 
 php:
   - 7.2
+  - 7.4snapshot
```
Comment (dereuromark) [inline .travis.yml]: 7.4 please

Comment (dereuromark) [inline .travis.yml]: 7.4 please now

**Comment by dereuromark** [comment]
Any update here? We should finish and merge this.

**Comment by dereuromark** [comment]
ping @ehsanmx

**Comment by RodinVasily** [comment]
Aha feature: https://spryker.aha.io/features/TECH-350

**Pull request closed without merging**